README.md

```diff
@@ -3,23 +3,47 @@ Example CNF
+Example CNF is an OpenShift workload to exercice an SRIOV setup, such as the following:
 
 ![Schema](documentation/schema.png)
 
 It is providing the following operators:
 
-* testpmd-operator
 * trex-operator
+    * It provides TRex Traffic Generator, decomposed in three components:
+        * TRexConfig CR, pinning the config of the TRex server instance. Related pod is `trexconfig-<x>` pod.
+        * TRexApp CR, setting up the job that launches TRex execution to generate traffic towards the system under test. Related pod is `trex-app` pod.
+        * TRex Operator, ensuring CR reconciliation via controller-manager pod. Related pod is `trex-operator-controller-manager-<x>` pod.
+    * Following information can be extracted from pod logs:
+        * In `trexconfig-<x>` pod logs, you can see the trex statistics printed periodically.
+        * The summary of the test execution can be seen at the end of the `trex-app` job logs.
+        * In `trex-operator-controller-manager-<x>` pod, you can see the execution of the Ansible playbooks that ensures the reconciliation loop of the operator.
 * testpmd-lb-operator
+    * It represents a modified TestPMD (Packet Manipulation Daemon) instance, called TestPMD LoadBalancer (LB), implementing a custom load balancing forwarding module, using two components:
+        * LoadBalancer CR, creating the pod that perform load balancing between its ports. Related pod is `loadbalancer-<x>` pod. This pod is composed by two containers: `loadbalancer`, which performs the load balancing forwarding, and `listener`, an auxiliary module that is listening to the CNFAppMac component created by the cnf-app-mac-operator to retrieve the MAC addresses of the TestPMD instances launched by the testpmd-operator (i.e. the CNF Application), then serving this information to the `loadbalancer` container.
+        * TestPMD LB Operator, ensuring CR reconciliation via controller-manager pod. Related pod is `testpmd-lb-operator-controller-manager-<x>` pod.
+    * Following information can be extracted from pod logs:
+        * To see the TestPMD LB statistics printed periodically for this module, you can rely on `loadbalancer-<x>` pod logs (which prints `loadbalancer` container logs).
+        * In `testpmd-lb-operator-controller-manager-<x>` pod, you can see the execution of the Ansible playbooks that ensures the reconciliation loop of the operator.
+    * The LoadBalancer is not currently replicated; only one instance is deployed. It's supposed that, up to now, it's not belonging to any update-upgrade process of the example-cnf, so that it has to be deployed in an isolated worker node, whereas the other components of this CNF (TRex and CNF Application) have to be deployed in a different worker node. Pod anti-affinity rules ensure this setup.
+* testpmd-operator
+    * Final application, also known as CNF Application, which is a standard TestPMD instance using the default MAC forwarding module. It uses two components:
```

Comment:
'the default MAC forwarding module'
Do you mean 'by default' or the regular module?